### DISCUSSION TITLE
[Jan 31, 2023] Vercel Dashboard, Builds, and API are experiencing degradation

### DISCUSSION THREAD

**Discussion by jgtvares** [vercel/community#1402] [Help] [answered]
### Summary

Suddenly I'm not able to load any page in my projects.
I'm catching a lot of `502` errors and foverer `pending` requests in my Network dev tools tab.

**Comment by jgtvares** [comment, marked as answer]
Yes, it is 😢 
https://www.vercel-status.com/

**Reply by leerob** [reply]
This has been resolved: https://www.vercel-status.com/incidents/j5h5zqjld4mm
